terraform/docker/Dockerfile.terrarium

```diff
@@ -189,5 +189,8 @@ RUN cd /opt \
     && BUNDLE_SILENCE_ROOT_WARNING=true bundle install --full-index --jobs=8 \
     && rm -Rf $HOME/.bundle/cache
 
+RUN curl -s https://raw.githubusercontent.com/aquasecurity/tfsec/master/scripts/install_linux.sh | bash
```

Comment:
Please be also careful in regards to Version constraints and cross platform builds as the install script might pick inappropriate target platform!